.clabot

```diff
@@ -1,5 +1,6 @@
 {
   "contributors": [
+    "actions-user",
     "expede"
```

Comment:
Also need to do the same for IPR in the `Notices.md`. The `.clabot` file is just the GitHub Actions helper (which annoyingly doesn't know about `Notices.md` because it's a general tool)